eth_tester/backends/pyethereum/v20/main.py

```diff
@@ -46,6 +47,17 @@
 from eth_tester.backends.pyethereum.validation import (
     validate_transaction,
 )
+from eth_tester.utils.formatting import (
+    replace_exceptions,
+)
+
+
+if is_pyethereum21_available():
+    from ethereum.tools.tester import (
+        TransactionFailed as Pyeth21TransactionFailed,
+    )
+else:
+    Pyeth21TransactionFailed = Exception
```

Comment:
Same comment here about setting to `None` instead.

Reply:
Oh oops, good call.